mistletoe/token.py

```diff
@@ -0,0 +1,47 @@
+"""
+Base token class
+"""
+
+import html
+import re
+import mistletoe.span_tokenizer as tokenizer
+from mistletoe import core_tokens
```

Comment:
all these imports don't seem to be used here

Reply:
Removed

Reply:
OK.